### DISCUSSION TITLE
Add GoogleMeet URL

### DISCUSSION THREAD

**Discussion by eximius313** [InteractionDesignFoundation/add-event-to-calendar-docs#38] [General]
Is there a way to add Google Meet URL so it looks linke this:
![image](https://user-images.githubusercontent.com/5578017/195195010-f591de12-7f32-4f0c-9195-d6ed920154b1.png)
?

In `ics` files Google uses `X-GOOGLE-CONFERENCE:https://meet.google.com/xyz-xyd` for this purpose

**Comment by alies-dev** [comment]
There is a parameter for this, `vcon=meet`:

```
https://calendar.google.com/calendar/render?action=TEMPLATE&text=Test&dates=20261231T190000Z/20261231T200000Z&vcon=meet
```

Verified on 2026-08-19: the editor opens with the "Join with Google Meet" block already filled in and a real link generated.

It does not do what the question asks for exactly, though. The client compares the value against the literal string `meet` and then asks the backend for a **new** conference, so you cannot hand it an existing `https://meet.google.com/xyz-xyd` URL. There is no URL parameter equivalent to `X-GOOGLE-CONFERENCE` in ICS.

If you need a specific pre-existing Meet link on the event, the options are an ICS file with `X-GOOGLE-CONFERENCE`, or the Calendar API with `conferenceData`. Through a plain hyperlink the closest you can get is putting the link in `details` or `location`.

Now documented in `services/google.md`.
